doc/QuickStart.md

```diff
@@ -29,23 +32,48 @@ Installing
 
 ### Prerequisites
 
-#### Installing CPAN modules
+#### Requirements *(root)*
+
+	$ dnf -y install perl perl-CPAN git screen crontabs nano
+> or 		$ aptitude install perl perl-CPAN git screen crontabs nano
+> or 		$ apt-get install perl perl-CPAN git screen crontabs nano
+
+	$ adduser -m pbot
+	
+
+#### Installing CPAN modules *(root)*
 
 PBot has many features; some of these depend on Perl modules written by others.
-This list can be found in the `MODULES` file in the root directory of this source.
+This list can be found in the [`MODULES`](https://github.com/pragma-/pbot/blob/master/MODULES) file in the root directory of this source.
 
 The modules may be installed with a simple command:
 
-    cpan -f -i $(cat MODULES)
+    $ cpan
+	cpan[1]> force install <modules> 
+	cpan[49]> exit
 
 Some CPAN modules may fail to pass certain tests due to outdated variables.
 Despite these test failures, their core functionality should still work as
 expected.
 
+#### Installing a crontab *(pbot)*
+
+	$ su pbot
+	$ crontab -e
+
+0,5,10,15,20,25,30,35,40,45,50,55 * * * * screen -dmS pbot /home/pbot/pbot/pbot data_dir=/home/pbot/pbot/freenode >/dev/null 2>&1
+
+>  or
+>   0,15,30,45 * * * * screen -dmS pbot2 /home/pbot/pbot/pbot data_dir=/home/pbot/pbot/ircnet >/dev/null 2>&1
+>> or
+> 0,20,40 * * * * screen -dmS coolbot /home/pbot/pbot/pbot data_dir=/home/pbot/pbot/coolbot >/dev/null 2>&1
+
+	
+
```

Comment:
Local operating system configuration is beyond the scope of this QuickStart guide. Users are expected to understand their system and how they wish to start PBot.